### ISSUE TITLE
Issues with task 939, 941, 943

### ISSUE THREAD

**Issue by tanay2001** [allenai/natural-instructions#599]
I just noticed after #595 was merged it changed the `output_language` label for tasks `939`,`941`,`943`. The outputs for the following tasks are meant to be in English based on how the json file was created.

Just wanna confirm if this is an issue or its fine 

Thanks

**Comment by danyaljj** [comment]
You're right, I missed your comment 🤦 
Let's fix these issues in a new PR.
